ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -414,22 +413,20 @@ spine:
       mgmt_ip: 172.16.1.12/24
 ```
 
-This time the settings defined are no longer fabric-wide, but are limited to DC1. In this example where we only have a single data center this is of limited benefit, but it allows us to scale our configuration to a scenario with multiple data centers in the future.
-
-Initially we define the default gateway for the management interface of all devices in DC1.
+Initially the default gateway for the management interface of all devices in DC1 is defined.
```

Comment:
Just to double check, the snippet above defines a `mgmt_ip` but not a `mgmt_gateway`. is this what is expected?

Reply:
mgmt_gateway is defined at the very top:

```yml
---
mgmt_gateway: 172.16.1.1
```